### DISCUSSION TITLE
eyepie vs eyepy

### DISCUSSION THREAD

**Discussion by drombas** [MedVisBonn/eyepy#13] [Ideas]
Hi!

I noticed that you use `eyepie` instead of `eyepy` I guess because there is already an [eyepy](https://pypi.org/project/eyepy/) repo on PyPI. That project seems to be totally staled.

I wonder if it could make sense to reach out to the developer and kindly ask to remove it so that we can use eyepy and avoid confusions.

**Comment by Oli4** [comment]
Hi @drombas 

I tried to contact @dillongwozdz the current ower of the eyepy name on PyPI via Email a while ago but didn't receive an answer. Maybe my request ended up in his spam. I took your hint here and now tried to contact him on LinkedIn. And maybe he can read this because of the mention above. If we can not contact him the process of transferring a PyPI name is described [here](https://peps.python.org/pep-0541/)

@dillongwozdz, if you can read this please let us know whether you would be willing to transfer the PyPI name "eyepy" to our project.

## Update 
Great news, I got an answer via LinkedIn and now I am the owner of the eyepy name on PyPI. I have to figure what's the best way for switching from eyepie to eyepy but soon this issue is going to be resolved. Thanks to @dillongwozdz for transferring the name!

**Comment by Oli4** [comment]
I just released a new version to PyPI under the `eyepy` package name. From now on for all versions >= 0.12.0 packages are going to be released under the new name. Prior versions are still available under the old name (`eyepie`)

**Reply by drombas** [reply]
Brilliant!
